blog/README.md

```diff
@@ -0,0 +1,232 @@
+# Red Hat blog
+
+In this blog we are going to explore the coreos-layering technology in order to
+inject kernel-modules to an Openshift node.
+
+### Prerequisits
+
+* OCP 4.12 or later.
+
+### Background
+
+This time we are going to build a container image that contains a kernel-module
+that we wish to load into the node's kernel.
+
+We will use the [driver-toolkit](https://github.com/openshift/driver-toolkit)
```

Comment:
I didn't want to use simple-kmod because:
* KMM is using DTK
* DTK is using simple-kmod instead of kmm-kmod to not create a loop dependency (DTK depending on KMM for it's CI module).

Therefore, I though it is better to use the simple-kmod when using DTK standalone to not involve KMM in it as well.

Reply:
Mentioning KMM as an alternative is also a good addition to the blog.

Reply:
Added a reference to KMM - https://github.com/ybettan/coreos-layering-driver-toolkit/pull/4/files#diff-78f236b87da98227ae670c36caae7662a3f9e664737f5a8feecdbd5d8b71545aR234-R239